adsingestp/parsers/crossref.py

```diff
@@ -342,10 +361,18 @@ def parse(self, text):
 
         self._parse_issue()
         self._parse_title_abstract()
-        self._parse_contrib()
+        try:
```

Comment:
Is this ever throwing an exception? If so, we should handle it within the `_parse_contrib` function (and log it, if necessary) and remove the try/except

Reply:
I think this was how I found issue #25 but I'll verify.